client/extensions/woocommerce/app/products/index.js

```diff
@@ -43,6 +57,11 @@ class Products extends Component {
 	// Total number of results per page (the API returns 10)
 	perPage = 10;
 
+	state = {
+		isSearching: false,
```

Comment:
It seems like is less `isSearching` in the sense that a query is running that we are waiting for and more like `isShowingSearchResults` - is that right? Would you consider renaming?